crates/turborepo-scm/src/package_deps.rs

```diff
@@ -9,6 +9,8 @@ use crate::{hash_object::hash_objects, Error, Git, SCM};
 
 pub type GitHashes = HashMap<RelativeUnixPathBuf, String>;
 
+const INPUT_INCLUDE_DEFAULT_FILES: &str = "$TURBO_DEFAULT";
```

Comment:
Can we make this token `$TURBO_DEFAULT$`? I am concerned that it will be confused with an environment variable this way. I could even see `$$TURBO_DEFAULT$$` being more clear.

Reply:
👍 for `$TURBO_DEFAULT$`.